pkg/controller/policydecision/policydecision_controller.go

```diff
@@ -513,6 +513,11 @@ func (r *ReconcilePolicyDecision) ingressForPolicyDecision(instance *operatorv1a
 
 func (r *ReconcilePolicyDecision) deploymentForPolicyDecision(instance *operatorv1alpha1.PolicyDecision) *appsv1.Deployment {
 
+	// Update the audit image for upgrade scenarios
+	if instance.Spec.AuditService.ImageName != "audit-syslog-service" {
+		instance.Spec.AuditService.ImageName = "audit-syslog-service"
```

Comment:
audit-syslog-service is being referenced in multiple places, should we create a constant variable to hold this value?

Reply:
added a constants.go file where we can hold variables like this